Makefile

```diff
@@ -12,5 +35,19 @@ updaterdk:
 	go get go.viam.com/rdk@latest
 	go mod tidy
 
+FFmpeg:
+	git clone https://github.com/FFmpeg/FFmpeg.git --depth 1 --branch release/6.1
+
+$(FFMPEG_PREFIX): FFmpeg
+	cd FFmpeg && ./configure $(FFMPEG_OPTS) && make -j$(shell nproc) && make install
+
+build-ffmpeg:
+ifeq ($(UNAME_S),Linux)
+ifeq ($(UNAME_M),x86_64)
+	which nasm || (sudo apt update && sudo apt install -y nasm)
```

Comment:
In amd64 antique container, we need to install nasm for the optimized assembly kernels to work.

Reply:
1. How do we know if a linux x86_64 executable of this module will use the optimized assembly kernels?

2. Does this put any requirements on the machine that runs the executable (libraries, OS version, kernel version, etc)?

Reply:
- ASM code should just be compiled in the library
- For example, check out h264 asm backend here: https://github.com/FFmpeg/FFmpeg/blob/8ca57fcf9ed327a6c2d9c5345be9b7e0724ca048/libavcodec/x86/h264_deblock.asm#L4

- Do not think there are additional considerations for kernel/os for asm stuff.
    - Any system calls should have the same requirements as from the go/c/cpp code.
- Just needed the `nasm` utility to build the asm code through ffmpeg builder.
        - If it cannot build assembly it will default to the unoptimized c code, so that it was important to include.